tools/gulp/util/task_helpers.ts

```diff
@@ -148,7 +148,6 @@ export function serverTask(packagePath: string, livereload = true) {
       root: projectDir,
       livereload: livereload,
       port: 4200,
-      fallback: path.join(packagePath, 'index.html'),
```

Comment:
Good point. I tried to accomplish this using a http rewrite rule. Also added comments now for those rewrite rules.